Jenkinsfile

```diff
@@ -16,7 +16,7 @@ pipeline {
             dir(path: 'test') {
               sh '''#!/bin/bash
                  . activate hls4ml-py36
-                 cat keras-models.txt | xargs ./keras-to-hls.sh -p 3'''
+                 ./convert-keras-models.sh -p 3 -f keras-models.txt'''
```

Comment:
Does this need to have `-x` as a command line arg? Just running this command did not generate converted output for me

Reply:
Good catch! I have fixed it in the latest commit.